src/mixins/validation.js

```diff
@@ -21,7 +21,9 @@ export const validation = {
         isValidDerivationPath: val => /([m]\/[0-9]{1,2}(')(\/)[0-9]{1,2}(')\/[0-9]{1,2}(')\/[0-9]('){1,30})/.test(val) || this.$t('forms.errors.isNotValidDerivationPath'),
         isValidMainetBTC: val => /([13][a-km-zA-HJ-NP-Z1-9]{25,34}|bc1[ac-hj-np-zAC-HJ-NP-Z02-9]{11,71})/.test(val) || this.$t('forms.errors.isNotValidMainetBTC'),
         isValidPolkadotAddress: val => this.$store.$polkadotApi?.isValidPolkadotAddress(val) || this.$t('forms.errors.isNotValidPolkadotAddress'),
-        notOwnAccount: value => val => !(value === val) || this.$t('forms.errors.notOwnAccount')
+        notOwnAccount: value => val => !(value === val) || this.$t('forms.errors.notOwnAccount'),
+        greaterOrEqualThanString: value => val => val.length >= value || this.$t('forms.errors.greaterOrEqualThanString', { value }),
```

Comment:
why we are duplicating this rule?

Reply:
My bad. Maybe I added when fixing the conflicts. I'll remove the duplicate line. Thanks!

Reply:
Sorry, about this, greaterOrEqualThanString only applies for `String` the other is used on `Numbers`, that's the difference

Reply:
> Sorry, about this, greaterOrEqualThanString only applies for `String` the other is used on `Numbers`, that's the difference

So we are using this rules to diff the length for a string? why is this need?